scripts/west_commands/runners/jlink.py

```diff
@@ -146,5 +158,10 @@ def flash(self, **kwargs):
                     '-device', self.device,
                     '-CommanderScript', fname])
 
+            if self.jtagconf:
```

Comment:
Can we just have  a single option that adds to the commander command line? I'm trying to avoid too many options that map 1:1 to underlying tool options.

Reply:
Are you thinking of something like `--additional-args='-jtagconf -1,-1 -autoconnect 1'`?

Reply:
More like pyocd.py's --flash-opt, an option with action='append' rather than commas or some other type of delimiter.

Some runner-related goals I have are:

- unify various runner-specific options using the RunnerCaps mechanism, thus providing common options depending on whether each runner can do some specific thing (like "chip erase" vs. "sector erase", "use this debugger port", "take this random identifier specifying exactly which connected board I want you to use"),
- said unification to include a common option, shared by all runners that want it, for extending the underlying runner tool's command line, that behaves like pyocd's `--flash-opt` does today

That will make flashing / debugging commands work the same for all boards based on features their debug probe supports, and not just the particular Zephyr runner they happen to use.

Haven't had time for v1.14 but that's coming up next and I'm trying to avoid adding additional options then deprecating them later, and this definitely qualifies.

I don't know what I'm going to call this option, but perhaps `--tool-opt` would be a good place to start unless you've got another idea.

Reply:
`--tool-opt` with `action='append'` 👍 I'll take a shot at it and see what you think